blueprints/cloud-operations/network-dashboard/main.tf

```diff
@@ -13,7 +13,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+     
+terraform {
+  required_version = "~> 1.3.3"
+  required_providers {
+    google = {
+      version = "~> 4.41.0"
+    }
+  }
+}
```

Comment:
We typically copy `default-versions.tf` at the root of the repo and put it in `versions.tf` inside the module/blueprint. Can you do that here too?

Reply:
Yes, just made the fixes.